packages/yarnpkg-core/sources/Project.ts

```diff
@@ -1347,7 +1347,7 @@ export class Project {
 
                 xfs.detachTemp(logDir);
 
-                const buildMessage = `${structUtils.prettyLocator(this.configuration, pkg)} couldn't be built successfully (exit code ${this.configuration.format(String(exitCode), FormatType.NUMBER)}, logs can be found here: ${this.configuration.format(logFile, FormatType.PATH)})`;
+                const buildMessage = `${structUtils.prettyLocator(this.configuration, pkg)} couldn't be built successfully (exit code ${this.configuration.format(String(exitCode), FormatType.NUMBER)}, logs can be found here: ${this.configuration.format(npath.fromPortablePath(logFile), FormatType.PATH)})`;
```

Comment:
I think we should just instead fix `configuration.format` to take care of that when the type is `FormatType.PATH`.

Reply:
That's a way better solution, fixed